apps/desktop-wallet/src/features/send/sentTransactions/SentTransactionSnackbarPopup.tsx

```diff
@@ -64,20 +67,21 @@ const SentTransactionSnackbarPopup = memo(({ txHash }: SentTransactionSnackbarPo
   return (
     <SentTransactionSnackbarPopupStyled {...fadeInBottom} {...fadeOut} className="info">
       <Columns>
+        <Progress status={sentTx.status} />
         <Rows>
-          <Message status={sentTx.status} />
-
-          <HashEllipsedStyled hash={txHash} tooltipText={t('Copy hash')} showSnackbarOnCopied={false} />
-
-          <Progress status={sentTx.status} />
+          <b>
+            <Message status={sentTx.status} />
+          </b>
```

Comment:
nit: Let's use `font-weight` on the `Message`, WDYT?

Reply:
The way you did it `Message` only returns a string, not an HTML component, so I didn't want to touch it.
But when I think about it again we should indeed return a component to follow our convention, or don't use uppercase on `Message`.

On it.

Reply:
Done